pyproject.toml

```diff
@@ -105,7 +105,7 @@ e2e-tests = "pytest test/e2e/"
 bump-version = """
     # get current version, append ".postN" (N=no. of commits since last release), write back into pyproject.toml
     version=$(git describe --tags --abbrev=0)
-    commit_cnt=$(git rev-list --count HEAD ^$version)
+    commit_cnt=$(git rev-list --count main ^$version)
```

Comment:
HEAD is the current branch. When a PR is merged into a PR, it counts the commit from the last release until the last commit of the PR-branch, which can be a too high number.